src/layout.rs

```diff
@@ -41,7 +41,7 @@ pub enum Constraint {
     /// assert_eq!(5, Constraint::Percentage(50).apply(11));
     /// ```
     Percentage(u16),
-    /// Apply a ratio  
+    /// Apply a ratio
```

Comment:
i dont know if this change was automatic, but removing the double space at the end will make the lines as if it were a single line, either re-add the double space, or add a empty line inbetween

Reply:
Let's use an empty line rather than invisible characters that affect things. The rust-doc convention is that the first line of a doc comment is a single sentence paragraph, not multiple lines with a line break.

Reply:
This has been resolved in another PR.